docs/docs/telemetry.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: telemetry
+sidebar_label: Rasa Telemetry
+title: Rasa Telemetry
+---
+
+Rasa uses telemetry to report anonymous usage information. This information is
```

Comment:
can you put this paragraph or at least part of it under `abstract:` below the title above? If you try doing `make livedocs` you'll see how that looks

Reply:
done :+1: